README.md

```diff
@@ -1,12 +1,45 @@
-![CI](https://github.com/nearform/bench-template/actions/workflows/ci.yml/badge.svg?event=push)
+![CI](https://github.com/nearform/fastify-constraints/actions/workflows/ci.yml/badge.svg?event=push)
 
-# Bench Template
+Fastify plugin to add constraints to multiple routes
 
-A feature-packed template to start a new repository on the bench, including:
+## Install
 
-- code linting with [ESlint](https://eslint.org) and [prettier](https://prettier.io)
-- pre-commit code linting and commit message linting with [husky](https://www.npmjs.com/package/husky) and [commitlint](https://commitlint.js.org/)
-- dependabot setup with automatic merging thanks to ["merge dependabot" GitHub action](https://github.com/fastify/github-action-merge-dependabot)
-- notifications about commits waiting to be released thanks to ["notify release" GitHub action](https://github.com/nearform/github-action-notify-release)
-- PRs' linked issues check with ["check linked issues" GitHub action](https://github.com/nearform/github-action-check-linked-issues)
-- Continuous Integration GitHub workflow
+```bash
+npm i fastify-constraints
+```
+
+## Usage
+
+Register fastifyConstraints as a Fastify plugin.
```

Comment:
let's name the plugin consistently, it's called fastify-constraints. Also, consider wrapping it in backticks to format it as code